<launch>
  <node pkg="lab_usb_9axisimu_driver" name="lab_usb_9axisimu_driver" type="lab_usb_9axisimu_driver">
    <param name="port" value="/dev/ttyACM0"/>
  </node>
</launch>
